_posts/2015-12-11-License-and-You.markdown

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: 'Licenses for OSS Code'
+date: 2015-12-10T00:00:00.000Z
+comments: false
+categories: [ios, mobile, review, video, oss]
+author: orta
+---
+
+As a part of our internal Lunch and Learn series, I  gave a talk to our developers with an overview of the different types of source code licenses available.
+
+We always recommend MIT within Artsy, but understanding what the other ones are and why we made the choices we have done is valuable.
+
+Jump [to YouTube](https://www.youtube.com/watch?v=0r7JcN3Q_LY) for the video, or click more for a smaller inline preview, as well as all of the speakers notes.
+
+<!-- more -->
+
+{% youtube 0r7JcN3Q_LY %}
+
+### Licenses For Code
+
+Attn:*I am not a lawyer*, but I have to care about this from an app dev perspective, and a dependency manager perspective.
+
+Recommendation for further reading:
+
+* http://choosealicense.com
+* https://tldrlegal.com/
+
+We'll talk about 3 types of licenses:
+
+- Permissive: MIT/BSD - Jquery/ Rails/ Artsy OSS
+- Mostly Permissive: Apache, Mozilla - SVN / Firefox
+- Copyleft / Viral - Git/ Linux/ Wordpress
+
+
+### Common Components
+
+All Share one important thing:
+
+  * YOU MAY USE THIS
+  * NO WARRANTY
+  * DISCLAIMER OF LIABILITY
+
+Which is basically the crux of how we can all actually use and write OSS without ending up bankrupt. E.g. when a major component of your infrastructure "fails" - it's not the fault of the author. Example: [Shellshock](https://en.wikipedia.org/wiki/Shellshock_(software_bug)) - no-one tried suing the maintainers of Bash.
+
+### Public Domain
+
+By not including a license you are specifically letting your code only applying copyright to the code. All code is the copyright of someone. Copyright is, err, complex. Roughly speaking though, there is no contract between you and someone using your code. This means _all_ of the power is in favour of the library author. A library consumer would need to email the author and ask for permission on a case by case basis, even then there is nothing stopping a developer revoking the ability for you to use their library after the fact. It's also different depending on the country the library author is in. However in the end PD is not a license, so add one.
+
+
+[This is basically the default license for code. ](http://blog.codinghorror.com/pick-a-license-any-license/)
+
+This means that code on a blog could not be safe to use unless they've declared its license, whereas for something like stack overflow all of the contents there are licensed under the Creative Commons. Which I'll talk about later.
```

Comment:
Could be worth linking to SO's blog post about using CC: http://blog.stackoverflow.com/2009/06/attribution-required/